### PR TITLE
wgsl: Add AF `select` execution tests

### LINKED ISSUE #3076: wgsl: Add AbstractFloat `select` execution tests
Going through the list of tests I need to write, and noticed this one doesn't have an obvious issue for it. Probably should get it done, since select is kinda a load bearing builtin for the AF tests...

### PR DESCRIPTION
**Pull request by zoddicus**
Fixes #3076

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

**Review by jzm-intel — APPROVED** [review]
Looks good to me, thanks

File: src/webgpu/shader/execution/expression/call/builtin/select.spec.ts [hunk 1]
```diff
@@ -133,7 +140,7 @@ g.test('scalar')
 
     await run(
       t,
-      builtin('select'),
+      t.params.component !== 'af' ? builtin('select') : abstractBuiltin('select'),
```
Comment (dneto0) [inline src/webgpu/shader/execution/expression/call/builtin/select.spec.ts:143]: nit.  I would flip the condition around and rephrase it in the positive sense.

**Review by dneto0 — APPROVED** [review]

File: src/webgpu/shader/execution/expression/call/builtin/select.spec.ts (hunk 1 above)
Comment (dneto0) [reply src/webgpu/shader/execution/expression/call/builtin/select.spec.ts:143]: there's a similar one later too.

not a blocking request

Comment (zoddicus) [reply src/webgpu/shader/execution/expression/call/builtin/select.spec.ts]: Done

**Pull request merged**